titus-server-runtime/src/main/java/com/netflix/titus/runtime/connector/relocation/client/GrpcAdapterRelocationServiceClient.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2018 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.runtime.connector.relocation.client;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import com.netflix.titus.api.relocation.model.TaskRelocationPlan;
+import com.netflix.titus.common.util.StringExt;
+import com.netflix.titus.grpc.protogen.Page;
+import com.netflix.titus.grpc.protogen.TaskRelocationQuery;
+import com.netflix.titus.runtime.connector.relocation.ProtobufRelocationServiceClient;
+import com.netflix.titus.runtime.connector.relocation.RelocationServiceClient;
+import com.netflix.titus.runtime.relocation.endpoint.RelocationGrpcModelConverters;
+import reactor.core.publisher.Mono;
+
+import static com.netflix.titus.runtime.relocation.endpoint.RelocationGrpcModelConverters.toCoreTaskRelocationPlan;
+
+/**
+ * {@link RelocationServiceClient} implementation that translates all invocation into underlying GRPC calls.
+ */
+@Singleton
+public class GrpcAdapterRelocationServiceClient implements RelocationServiceClient {
+
+    private static final Page ONE_ITEM_PAGE = Page.newBuilder().setPageSize(1).build();
+
+    private final ProtobufRelocationServiceClient protobufClient;
```

Comment:
grpcClient?

Reply:
This is a wrapper around GRPC client, but probably this name is less confusing than the current one.